### PR TITLE
Implement fizzle for all cards that draw preludes.

### PR DESCRIPTION
**Pull request by kberg**
The only thing left is fizzle during standard prelude phase.

### CODE REVIEW

File: src/locales/nl/log_messages.json [hunk 1]
```diff
@@ -281,10 +281,9 @@
     "${0} cannot turn ${1} face down to gain 3 TR because of unaffordable Reds policy cost": "${0} kan ${1} niet omdraaien om 3 TW te krijgen vanwege de hoge kosten van het DE RODEN-beleid",
     "${0} turned ${1} face down to gain 3 TR": "${0} heeft ${1} omgedraaid om 3 TW te krijgen",
     "You drew ${0}": "Je trok ${0}",
-    "You drew ${0} and ${1}": "Je trok ${0} en ${1}",
+    "DELETE 2023-09-01 You drew ${0} and ${1}": "Je trok ${0} en ${1}",
```
Comment (nwai90) [inline src/locales/nl/log_messages.json:284]: Is this change intended?

Comment (kberg) [reply src/locales/nl/log_messages.json:284]: It was, but now I regret it.

**Review by nwai90 — APPROVED** [review]
Looks good!
